composer.json

```diff
@@ -89,7 +89,7 @@
         "orchestra/testbench-core": "^6.23",
         "pda/pheanstalk": "^4.0",
         "phpunit/phpunit": "^8.5.19|^9.5.8",
-        "predis/predis": "^1.1.8",
+        "predis/predis": "v1.1.x-dev",
```

Comment:
`^1.1.9`

Reply:
I'll do that when Till has actually tagged the release.

Reply:
why? it'll work even without, and it means we don't need to go back and update the code later.

Reply:
I'd agree with testing `v1.1.x-dev` first before I tag, just in case there is anther minor compat issue.